charts/adot-exporter-for-eks-on-ec2/values.yaml

```diff
@@ -331,11 +337,58 @@ adotCollector:
       - dimensions: [[Namespace, ClusterName], [ClusterName]]
         metric_name_selectors:
           - namespace_number_of_running_pods
+    ampreceivers:
+      scrapeInterval: 15s
+      scrapeTimeout: 10s
+      scrapeConfigs: |
+            - job_name: 'k8s_metrics_scrape'
+              sample_limit: 10000
+              metrics_path: /metrics
+              kubernetes_sd_configs:
+                - role: pod
+              relabel_configs:
+                - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+                  action: keep
+                  regex: true
+                - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+                  action: replace
+                  regex: (.+)
+                  target_label: __metrics_path__
+                - source_labels: [__address__]
+                  action: replace
+                  regex: ([^:]+)(?::\d+)?;(\d+)
+                  replacement: $1:$2
+                  target_label: __address__
+                - action: labelmap
+                  regex: __meta_kubernetes_pod_label_(.+)
+                - source_labels: [__meta_kubernetes_pod_node_name]
+                  action: keep
+                  regex: K8S_NODE_NAME
+                - source_labels: [__meta_kubernetes_namespace]
+                  action: replace
+                  target_label: K8S_NAMESPACE
+                - source_labels: [__meta_kubernetes_pod_name]
+                  action: replace
+                  target_label: K8S_POD_NAME
+                - source_labels: [__meta_kubernetes_pod_container_name]
+                  action: replace
+                  target_label: EKS_Container
+
+    ampexporters:
+      namespaces: "AMP data"
+      endpoint: "[Add Prometheus workspace remote write URL]"
+      resourcetotel: true
+      region: "[Add AWS Region]"
+      service: "aps"
```

Comment:
Is this necessary?  There shouldn't be any other `service` value used, I don't think.

Reply:
Not sure, this was present in the sample config for the AMP pipeline that I used. I'll remove the variable from values.yaml if it dosent make sense.